plugins/out_loki/loki.c

```diff
@@ -485,6 +486,31 @@ static int parse_labels(struct flb_loki *ctx)
     return 0;
 }
 
+static int prepare_remove_keys(struct flb_loki *ctx) {
+    int ret;
+    struct mk_list *head;
+    struct flb_slist_entry *entry;
+
+    /* Add remove keys set in the configuration */
+    if (ctx->remove_keys) {
+        struct flb_hash *ht = flb_hash_create(FLB_HASH_EVICT_NONE, 64, -1);
```

Comment:
Any suggestion on the size of a static hash table? In this case, we want some slots to be empty to quickly skip unmatched keys without another `strncmp()`, but of course this would waste some memory.

Reply:
However in most cases, `key_len` comparison and `strncmp()` are fast enough. Maybe I shouldn't worry about this and just need a generic size recommendation function for static hash tables.

In the Record Modifier Filter plugin, an `mk_list` is used instead of a hash table. Does it have better performance when there are not many keys in the list? Should I switch to that instead?